internal/tfimport/importer/simple_importer.go

```diff
@@ -15,39 +15,24 @@
 package importer
 
 import (
-	"bytes"
-	"html/template"
-
 	"github.com/GoogleCloudPlatform/healthcare-data-protection-suite/internal/terraform"
 )
 
 // SimpleImporter defines a struct with the necessary information to import a resource that only depends on fields from the plan.
 // Should be used by any resource that doesn't use interactivity, API calls, or complicated processing of fields.
 type SimpleImporter struct {
-	Fields []string
-	Tmpl   string
+	Fields   []string
+	Tmpl     string
+	Defaults map[string]interface{}
 }
 
 // ImportID returns the ID of the resource to use in importing.
 func (i *SimpleImporter) ImportID(rc terraform.ResourceChange, pcv ProviderConfigMap, interactive bool) (string, error) {
 	// Get required fields.
-	fieldsMap, err := loadFields(i.Fields, interactive, rc.Change.After, pcv)
-	if err != nil {
-		return "", err
-	}
-
-	// Build the template.
-	buf := &bytes.Buffer{}
-	tmpl, err := template.New("").Option("missingkey=error").Parse(i.Tmpl)
-	if err != nil {
-		return "", err
-	}
-
-	// Execute the template.
-	err = tmpl.Execute(buf, fieldsMap)
+	fieldsMap, err := loadFields(i.Fields, interactive, rc.Change.After, pcv, i.Defaults)
```

Comment:
Add a bit more documentation for how rc.Change.After, pcv and i.Defaults work together.